libraries/AP_BattMonitor/AP_BattMonitor.h

```diff
@@ -82,6 +82,13 @@ class AP_BattMonitor
         Critical
     };
 
+    // power states
+    enum class PowerState : uint8_t {
```

Comment:
```suggestion
    enum class ChargingState : uint8_t {
```

do we need an `UNKNOWN`?